src/queue/DefaultEventQueue-spec.ts

```diff
@@ -3,25 +3,36 @@ import { IEvent } from '../models/IEvent';
 import { expect } from 'chai';
 
 describe('DefaultEventQueue', () => {
+
+  let config: Configuration;
+
+  beforeEach(() => {
+    config = getConfiguration();
+  });
+
+  afterEach(() => {
+    let queue = <any>config.queue;
+    clearInterval(queue._queueTimer);
```

Comment:
This was needed to make all tests actually terminate and not run any code after the test has finished.